holoclean.py

```diff
@@ -216,14 +221,22 @@ def repair_errors(self, featurizers):
         print(status)
         if self.env['verbose']:
             print('Time to store repaired dataset: %.2f secs' % time)
+        if self.env['print_fw']:
+            status, time = self.repair_engine.get_featurizer_weights()
+            print(status)
```

Comment:
You'll need to update these `print` statements to using `logging` as in https://github.com/HoloClean/holoclean/pull/25.